test/Microsoft.ServiceFabric.Actors.Tests/Diagnostics/ActorFrameworkEventSourceTest.cs

```diff
@@ -22,8 +21,8 @@ public sealed class ActorFrameworkEventSourceTest : IDisposable
         // Test fixture
         static readonly IFuzz fuzzy = new RandomFuzz(Environment.TickCount);
 
-        public ActorFrameworkEventSourceTest(ITestOutputHelper output) =>
-            test = new EventSourceTest<ActorFrameworkEventSource>(output);
+        public ActorFrameworkEventSourceTest() =>
```

Comment:
Remove unnecessary constructor and initialize the instance variable inline.
Do this in all files where the ITestOutputHelper was removed.